projects/smoke_steering/src/main.c

```diff
@@ -36,9 +36,9 @@
 
 typedef enum {
   STEERING_DIGITAL_INPUT_HORN = 0,
-  STEERING_DIGITAL_INPUT_LANE_ASSIST,
-  STEERING_DIGITAL_INPUT_HIGH_BEAM_FORWARD,
-  STEERING_DIGITAL_INPUT_HIGH_BEAM_REAR,
+  STEERING_DIGITAL_INPUT_RADIO_PTT,
+  STEERING_DIGITAL_INPUT_DRL_1,
```

Comment:
I might be mistaken but I was told that moving the stalk forward would turn on high beams and moving it back would turn it off. It seems there are two daytime running lights here?

Reply:
The new requirements are that moving the stalk forward turns on DRL and moving it back turns them off - this was actually written before those requirements came out. This naming is already present in steering and this is just to keep smoke_steering consistent.